src/main/java/net/glowstone/block/blocktype/BlockType.java

```diff
@@ -293,29 +294,42 @@ public void onNearBlockChanged(GlowBlock block, BlockFace face, GlowBlock change
      * or {@link GlowBlock#setData} is called with physics enabled, and might
      * be called from plugins or other means of changing the block.
      *
-     * @param block The block that was changed
+     * @param block   The block that was changed
      * @param oldType The old Material
      * @param oldData The old data
      * @param newType The new Material
-     * @param data The new data
+     * @param data    The new data
      */
     public void onBlockChanged(GlowBlock block, Material oldType, byte oldData, Material newType,
-        byte data) {
+                               byte data) {
         // do nothing
     }
 
     /**
      * Called when the BlockType should calculate the current physics.
+     * <br/>
+     * <br/>
+     * Subclasses <b>SHOULD NOT</b> override this method,
```

Comment:
Remove those brs and use `<p>`. Also, you can just make the method final to prevent that.